frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineDetails.tsx

```diff
@@ -82,92 +80,96 @@ const PipelineDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath }) =
+                  selection={pipelineVersion?.display_name}
+                  onSelect={(version) =>
+                    navigate(
+                      routePipelineDetailsNamespace(
+                        namespace,
+                        version.pipeline_id,
+                        version.pipeline_version_id,
+                      ),
+                    )
+                  }
                 />
-              }
-              {...(pipelineVersion && {
-                description: (
-                  <MarkdownView
-                    component="span"
-                    conciseDisplay
-                    markdown={pipelineVersion.description}
+              </FlexItem>
+              <FlexItem>
+                {isLoaded && (
+                  <PipelineDetailsActions
+                    onDelete={() => setDeletionOpen(true)}
+                    pipeline={pipeline}
+                    pipelineVersion={pipelineVersion}
                   />
-                ),
-              })}
-              empty={false}
-              loaded={isLoaded}
-              headerAction={
-                isPipelineVersionLoaded && (
-                  <Flex
-                    spaceItems={{ default: 'spaceItemsMd' }}
-                    alignItems={{ default: 'alignItemsFlexStart' }}
-                  >
-                    <FlexItem style={{ width: '300px' }}>
-                      <PipelineVersionSelector
-                        pipelineId={pipeline?.pipeline_id}
-                        selection={pipelineVersion?.display_name}
-                        onSelect={(version) =>
-                          navigate(
-                            routePipelineDetailsNamespace(
-                              namespace,
-                              version.pipeline_id,
-                              version.pipeline_version_id,
-                            ),
-                          )
-                        }
-                      />
-                    </FlexItem>
-                    <FlexItem>
-                      {isLoaded && (
-                        <PipelineDetailsActions
-                          onDelete={() => setDeletionOpen(true)}
-                          pipeline={pipeline}
-                          pipelineVersion={pipelineVersion}
-                        />
-                      )}
-                    </FlexItem>
-                  </Flex>
-                )
-              }
-            >
+                )}
+              </FlexItem>
+            </Flex>
+          )
+        }
+      >
+        <PageSection
+          isFilled
+          padding={{ default: 'noPadding' }}
+          style={{ flexBasis: 0, overflowY: 'hidden' }}
+          variant="light"
+        >
+          <Flex
+            direction={{ default: 'column' }}
+            style={{ height: '100%' }}
+            spaceItems={{ default: 'spaceItemsNone' }}
+          >
+            <FlexItem>
```

Comment:
this flex and page section dont do anything

Reply:
same thing on the run details too

Reply:
Thank you @Gkrumbach07 removed the `Flex` components; `PageSection` is needed for the tabs to persist when the drawer is opened

Reply:
These Flex components are needed in order to size the graph area correctly such that the side panel scrolls and the graph doesn't expand below the height of the window which then puts the graph controls off screen.